Library/Util.ts

```diff
@@ -13,7 +13,7 @@ class Util {
             var cookieName = name + "=";
             var cookies = cookie.split(";");
             for (var i = 0; i < cookies.length; i++) {
-                var cookie = cookies[i];
+                var cookie: any = cookies[i];
```

Comment:
it's causing a build error for me. Probably different typescript version. I've removed it